tools/db_bench_tool.cc

```diff
@@ -4580,7 +4580,8 @@ void VerifyDBFromDB(std::string& truth_db_name) {
       if (FLAGS_max_scan_distance != 0) {
         if (FLAGS_reverse_iterator) {
           GenerateKeyFromInt(
-              (uint64_t)std::max((int64_t)0, seek_pos - FLAGS_max_scan_distance),
+              (uint64_t)std::max((int64_t)0,
+                                 seek_pos - FLAGS_max_scan_distance),
```

Comment:
This change is unrelated to this PR and got added by `make format`. But I think it might be ok to keep it in this PR as it is just a minor formatting fix.